src/errors/convert.js

```diff
@@ -54,5 +56,5 @@ export const convertAxiosError = (axiosError) => {
 		return axiosError
 	}
 
-	return new map[response.data.data.type]()
+	return new map[response.data.data.type](response.data.message)
```

Comment:
@ChristophWurst is it okay to forward the response message to the exception?

Reply:
sure!